sdk/communication/azure-communication-common/CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Release History
 
+## 1.1.0-beta.1 (Unreleased)
```

Comment:
This changelog change should be added to CHANGELOG files in other communication packages too.